includes/blocks/class-sensei-blocks.php

```diff
@@ -82,6 +82,9 @@ public function register_generic_assets() {
 
 		Sensei()->assets->register( 'sensei-blocks-frontend', 'blocks/frontend.js', [], true );
 		Sensei()->assets->register( 'sensei-theme-blocks', 'css/sensei-theme-blocks.css' );
+
+		wp_register_script( 'sensei-youtube-iframe-api', 'https://www.youtube.com/iframe_api', [], 'unversioned', true );
+		wp_register_script( 'sensei-vimeo-iframe-api', 'https://player.vimeo.com/api/player.js', [], 'unversioned', false );
```

Comment:
I changed to load the Vimeo in the `head` since the [documentation](https://developer.vimeo.com/player/sdk/basics#accessing-the-player-sdk) recommends adding it before our script.